src/aiperf/post_processors/otel_metrics_results_processor.py

```
@@ -0,0 +1,326 @@
+            timeout_millis=self._export_timeout_millis,
+        )
+
+    async def summarize(self) -> list[MetricResult]:
+        return []
+
+    @on_stop
+    async def _flush_and_shutdown(self) -> None:
+        """Final flush before shutdown and close SDK resources."""
+        try:
+            await self.flush(force=True)
+        except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)